src/anm/graphics/path.js

```diff
@@ -580,20 +541,38 @@ var encodeVisitor = function(segment, buffer) {
 Path.parse = function(path, target) {
     target = target || new Path();
     target.segs = [];
-    Path.visitStrPath(path, parserVisitor, target);
+    var segPaths = path.match(/[a-z][^a-z]*/ig);
```

Comment:
Is this way faster? Just asking — it is not very important in general, since path parsing only performed on import. Btw you may store `/[a-z][^a-z]*` in a `RegEx` object and re-use it like a constant.